libs/components/forms/src/lib/modules/checkbox/checkbox-label-text-label.component.html

```diff
@@ -0,0 +1,3 @@
+<span *ngIf="!labelHidden" class="sky-switch-label"
+  ><span skyTrim><ng-content /></span>
```

Comment:
Since we now control the text before it goes into the template, remove the `<span skyTrim>` element as its method of removing whitespace around text is inefficient (it watches the DOM for changes). There's also currently no use case for anything other than the `labelText` value being placed inside this component, so make `labelText` an input. Then you can trim the string before it's rendered, either with an Angular pipe or a setter in the component. Just be sure you continue to leave no empty spaces in any of the elements.